src/core/server/saved_objects/migrations/model/helpers.test.ts

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+import { addMustNotClausesToQuery } from './helpers';
+
+describe('addMustNotClausesToQuery', () => {
+  it('creates a valid query object if the existing query was not defined', () => {
+    const query: QueryDslQueryContainer = {};
+    const mustNotClauses = [{ term: { type: 'bar' } }, { term: { type: 'baz' } }];
+
+    expect(addMustNotClausesToQuery(query, mustNotClauses)).toMatchInlineSnapshot(`
```

Comment:
NIT/optional: I would use `toEqual` instead of `toMatchInlineSnapshot` here to increase readability of the assertions